addons/web_editor/static/lib/odoo-editor/test/spec/html_tables/google_odoo.html

```diff
@@ -0,0 +1,38 @@
+<table class="table table-bordered">
+        
+            
+            
+        
+        <tbody>
+            <tr>
+                <td><span style="font-weight: normal;font-style: italic;color: rgb(73, 80, 87)">
+                    <span style="font-size:10pt;font-style:italic;color:#495057">Italic then also
+                    </span><span style="font-size:10pt;font-weight:bold;font-style:italic;color:#495057">BOLD</span>
+                </span></td>
+                <td><span style="font-style: italic;text-decoration: line-through;color: rgb(73, 80, 87)">Italic strike</span></td>
+            </tr>
+            <tr>
+                <td><span style="font-weight: bold;color: rgb(73, 80, 87)">
+                    <span style="font-size:10pt;font-weight:bold;color:#495057">Just
+                        Bold </span><span style="font-size:10pt;font-style:italic;color:#495057">Just
+                        Italic</span>
+                </span></td>
+                <td><span style="font-weight: bold;text-decoration: underline;color: rgb(73, 80, 87)">Bold underline</span></td>
+            </tr>
+            <tr>
+                <td><span style="color: rgb(255, 0, 0)">Color text</span></td>
+                <td><span style="text-decoration: underline line-through;color: rgb(255, 0, 0)">Color
+                    strike and underline</span></td>
+            </tr>
+            <tr>
+                <td><span style="font-weight: normal;color: rgb(73, 80, 87)">Color background
+                </span></td>
+                <td><span style="color: rgb(255, 0, 0)">Color
+                    text on color background</span></td>
+            </tr>
+            <tr>
+                <td><span style="font-size: 14pt;font-weight: normal">14pt MONO TEXT</span></td>
+            </tr>
+        </tbody>
+    </table><p>
+[]</p>
```

Comment:
Are those `[]` really part of the test file though ? It feels strange as we usually use them to represent the selection in test cases but I'm not sure they belong here. Maybe I'm just missing something ? They are in a few files.

Reply:
The files ending in _odoo are the representation of the HTML inside the odoo editor. They're used for `contentAfter` and therefore must include the cursor position.

Reply:
Right, I didn't realize the naming convention when writing this comment, I realized it afterwards... ^^'